tests/acceptance/features/apiUserLDAPSharing/sharingLocalUserLdapUser.feature

```diff
@@ -0,0 +1,125 @@
+@api
+Feature: Sharing between local and LDAP users
+  As a user
+  I want to be able to share files and folders with any user regardless of the backend
+
+  Background:
+    Given user "local-user" has been created with default attributes in the database user backend
+    And these users have been initialized:
+    #this are LDAP users and are not initialized yet
+      | username | password |
+      | user0    | 123456   |
```

Comment:
why do we need to put the password in this table? 
doesn't the code underneath already know the "default" password that goes with each user?

Reply:
this particular does not have the magic, I will have to make a PR to core to change it

Reply:
PR here https://github.com/owncloud/core/pull/34213 , @phil-davis please review